app/src/sharedTest/java/org/oppia/app/player/state/StateFragmentTest.kt

```diff
@@ -299,6 +300,92 @@ class StateFragmentTest {
     }
   }
 
+  @Test
```

Comment:
Bunch of 100 char per line limit ktlint issue, please check with the recommended command.

Reply:
there are certain line count issues in the file should I change all of them? and I can't find line count issue at 303
i mean ktlint --android is not giving at line 303

Reply:
ok, you can fix at other lines.

Reply:
Done